lib/tasks/build.js

```diff
@@ -3,13 +3,12 @@
 const path = require('path');
 const merge = require('merge-stream');
 const BowerPlugin = require('bower-webpack-plugin');
-const webpack = require('webpack-stream');
+const webpack = require('webpack');
+const webpackStream = require('webpack-stream');
```

Comment:
webpack was already in package.json, so presumably being required somewhere else already. But either way, I just checked the webpack-stream api and it does expose the webpack library as a property so will change to use this instead